### PR TITLE
Sign in configuration

### PR DESCRIPTION
**Pull request by andresg4**
## TASKS
- [X] Created sessions controller to manage login response messages.

### CODE REVIEW

**Review by Alxzu — APPROVED** [review]
Nice!

File: app/views/api/v1/sessions/create_success.json.jbuilder [hunk 1]
```diff
@@ -0,0 +1,5 @@
+json.id @resource.id
```
Comment (jdrosales17) [inline app/views/api/v1/sessions/create_success.json.jbuilder]: Just a little thing here, you can align all the `@resource...` to make it more readable.

File: spec/requests/api/v1/sign_in_spec.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  let(:params)       { FactoryBot.attributes_for(:user) }
+  let(:params_login) { { email: params[:email], password: params[:password] } }
+  before(:each, confirm_email: true) do
+    sign_up
+  end
+
+  context 'signs in successfully', confirm_email: true do
+    it 'returns status 200 OK' do
+      post user_session_path, params: params_login, as: :json
+      expect(response.status).to eq(200)
```
Comment (jdrosales17) [inline spec/requests/api/v1/sign_in_spec.rb]: Avoid using plain numbers here, you can use status code symbols (in this case `:ok`).

File: spec/requests/api/v1/sign_in_spec.rb [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      expect(response.has_header?('access-token')).to eq(true)
+    end
+  end
+
+  context 'signs in fails' do
+    context 'wrong password', confirm_email: true do
+      it 'returns status 401 Unauthorized' do
+        params[:password] = '123'
+        post user_session_path, params: params_login, as: :json
+        expect(response.status).to eq(401)
```
Comment (jdrosales17) [inline spec/requests/api/v1/sign_in_spec.rb]: Same here (use status code symbol)

**Review by jdrosales17 — APPROVED** [review]
Nice work! Just some minor comments.

**Pull request merged**